less/components/jtable.less

```diff
@@ -81,6 +81,11 @@ a.JTable-row--href {
     padding-left: @table-column-spacing / 2;
     padding-right: @table-column-spacing / 2;
 
+    a&, a&:hover {
```

Comment:
It's stupid having to do these tricks to increase specificity. Almost as stupid as having to do all this work because you can't wrap a `<tr>` with `<a>`